personio-personnel-data-api.yaml

```
@@ -480,6 +480,22 @@ paths:
           description: >-
             Last day of the period to be queried.
             It is inclusive, so the day specified as end_date will also be considered on the results.
+        - name: updated_from
+          in: query
+          required: false
+          type: string
+          format: datetime
+          description: >-
+            First day of the period to be queried on which it could have been updated.
```

Just a suggestion, but maybe I would rephrase this as follow:
```suggestion
            The date-time of the period to be queried on which it could have been updated.
```
Sounds clearer to me. If this is how it is stated in the Absence one, then let's keep it consistent.